doc/dev/credscan_process.md

```diff
@@ -0,0 +1,80 @@
+# Guide for monitoring CredScan checks
+
+This guide describes how package owners can monitor their package's Credential Scanner (CredScan) status and correct
+any warnings.
+
+General information about CredScan can be found in the overview documentation at [aka.ms/credscan][credscan_doc]. The
+Azure SDK's motivation and methodology for running CredScan is documented [here][devops_doc].
+
+## Table of Contents
+- [Check CredScan status](#check-credscan-status)
+- [Correct active warnings](#correct-active-warnings)
+  - [True positives](#true-positives)
+  - [False positives](#false-positives)
+- [Correct baselined warnings](#correct-baselined-warnings)
+
+## Check CredScan status
+
+CredScan is run each night over the entire `azure-sdk-for-python` repository as part of the
+[python - aggregate-reports][aggregate_reports] pipeline. The scan produces a list of active warnings in the "Post
+Analysis" task of the "ComplianceTools" job ([example output][credscan_output]).
+
+Each warning will begin with the path to the file containing a potential credential, as well as the row and column where
+the credential string begins. For example, for a potential credential that starts in row 3 and column 20 of a
+particular file:
+```
+##[error]sdk/{service}/{package}/{file}.py:sdk/{service}/{package}/{file}.py(3,20)
+```
+
+The warning will then list an error code and description of why the potential credential was flagged.
+
+## Correct active warnings
+
+If you find any warnings listed for a package you own, the next step is to determine if the potential credential found
+by CredScan is an actual credential (a true positive), or a fake credential/false flag (a false positive).
+
+### True positives
+
+If CredScan discovers an actual credential, please contact the EngSys team at azuresdkengsysteam@microsoft.com so any
+remediation can be done.
+
+### False positives
+
+If CredScan flags something that's not actually a credential or secret, we can suppress the warning to shut off the
+false alarm. CredScan allows you to suppress fake credentials by either suppressing a string value or by suppressing
+warnings for a whole file. Suppressing string values will disable warnings no matter where the string comes up during a
+scan, but is inefficient and inconvenient for lengthy strings. Suppressing warnings in a file is convenient for
+fake credential files, but strings in that file will still trigger warnings if present in another unsuppressed file.
+Files that contain more than just fake credentials shouldn't be suppressed.
+
+Credential warnings are suppressed in [eng/CredScanSuppression.json][suppression_file]. Suppressed string values are in
```

Comment:
It is recommended to try and reuse an existing placeholder instead of adding new ones.

Reply:
Agreed -- I hope that's clear in this doc. Do you think [this list](https://github.com/Azure/azure-sdk-for-python/pull/22750/files#diff-ac4e865888ea48bc7aa78d9b1f67a5b727cbcaa94ccda798ae292c6c9c6c8a3fR57) of suppression options is accurate, in terms of which options are preferable?

Reply:
Yes I think that does cover it but I guess I missed it reading through the other text. Perhaps it might be worth moving up those steps so it is nice summary and add the text after for more details about the specifics.